invenio_github/tasks.py

```diff
@@ -114,30 +114,37 @@ def _get_err_obj(msg):
             err['error_id'] = str(g.sentry_event_id)
         return err
 
+    def _release_rest_exception_handler(release, ex):
```

Comment:
These three handlers can be moved outside the function (e.g. at the very top of the file), and be public (i.e. without the `_` prefix), since they can be re-used e.g. by the custom handlers.